src/commands/login.ts

```diff
@@ -0,0 +1,84 @@
+import { Command } from "@oclif/core";
+import OAuthServer, { ACCOUNT_URL } from "../lib/auth/oauth-client";
+import open from "open";
```

Comment:
want to pair on rewriting this to yargs and adding a few tests? the relevant bits are in the v3 branch.